app/views/budgets/_budget.html.erb

```diff
@@ -1,2 +1,2 @@
-<li><%= t("budgets.defaults.name") %>: <%= budget.name %></li>
-<li><%= t("budgets.defaults.budget_type") %>: <%= budget.budget_type %></li>
+<li><%= t("budgets.defaults.name") %>: <%= @budget.name %></li>
+<li><%= t("budgets.defaults.budget_type") %>: <%= @budget.budget_type %></li>
```

Comment:
Does budget type make sense now that there is only one budget per user?

Reply:
Probs not on reflection, I guess the name will say it all now, I will remove it in another PR